src/test/scala/org/broadinstitute/dsde/rawls/webservice/EntityApiServiceSpec.scala

```diff
@@ -798,6 +792,28 @@ class EntityApiServiceSpec extends ApiServiceSpec {
       }
   }
 
+  private def entityLessThan(attributeName: String)(e1: Entity, e2: Entity): Boolean = {
```

Comment:
I don't really care, sorting nulls first is probably in line with current behavior